dspy/dspy_default_config.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
Interesting file hmm.

Reply:
Same comment as above. Want a good way to serialize models, and save programs. For complicated pipelines, you need multiple LMs with different configs. Saving all of these in a config file and using them with a short-hand is very handy. 

We can not do this in DSPy and only do it in Irera, as a type of abstraction on top of DSPY. So, we can drop this here.

Still struggling with saving and loading LMs in DSPy though, that seems like a fundamental issue that should be handled in this repo.